### PR TITLE
tlv: fix typos, update example code in README.md

### PR DESCRIPTION
**Pull request by jacksondoherty**

### CODE REVIEW

File: libraries/type-length-value/README.md [hunk 1]
```diff
@@ -45,57 +44,63 @@ impl SplDiscriminate for MyOtherPodValue {
...
+    other_value1.data = 2;
+    
+    // Let's do it again, since we can now have repeating values!
+    let (other_value2, _) = state.init_value::<MyOtherPodValue>(true).unwrap();
+    assert_eq!(other_value2.data, 10);
+    // Update it in-place
+    other_value2.data = 4;
+    
+    // Later on, to work with it again, since we did _not_ allow repeating entries,
+    // we can just get the first value we encounter.
```
Comment (buffalojoec) [inline libraries/type-length-value/README.md:84]: Do you mind updating this comment to mention that specifically the `MyPodValue` type is the one we didn't allow repeats with? I think just rewording it is good.

**Review by buffalojoec — COMMENTED** [review]
Looks great, thanks! Just one request for updating the comment, even though I know you didn't add it!

File: libraries/type-length-value/README.md (hunk 1 above)
Comment (jacksondoherty) [reply libraries/type-length-value/README.md]: Done!

**Review by buffalojoec — APPROVED** [review]
Thanks!

**Pull request merged**